### DISCUSSION TITLE
Cannot disable enter to chat despite unchecking key remapping in runelite

### DISCUSSION THREAD

**Discussion by Shoruishi** [runelite/runelite#17921] [Q&A] [answered]
as the title suggests im stuck with enter to type in osrs for some reason, ive disabled key remapping both before and after unchecking the boxes on key remapping and tried re-launching runelite and restarting pc after doing so but nothing works, when i turn off key remapping the chat box goes empty but as soon as i try to type hit enter to type comes straight back again. driving me crazy.

**Comment by Alexsuperfly** [comment]
does this happen in safemode? toggle the respective box in the [RuneLite-Launcher-Configuration](https://github.com/runelite/runelite/wiki/RuneLite-Launcher-Configuration)

**Reply by Shoruishi** [reply]
no that seems to have worked actually, will runelite now default safe mode?

**Reply by Alexsuperfly** [reply, marked as answer]
safemode prevents hub plugins from loading, meaning this issue is caused by a hub plugin, you will have to try and find which one it is and uninstall it.  You can untick the safemode box to resume running normally.

**Reply by Shoruishi** [reply]
alright thanks dude :) appreciate it very much.

**Reply by Shoruishi** [reply]
got it, it was shift remapping separate to the menu entry swapper ones. thanks again!
